docs/data/material/guides/composition/composition.md

```diff
@@ -148,9 +86,9 @@ The other props of the `Typography` component will also be present in props of t
 
 You can find a code example with the Button and react-router-dom in [these demos](/material-ui/integrations/routing/#component-prop).
 
-#### Generic
+### Generic
 
-It's also possible to have a generic `CustomComponent` which will accept any React component, and HTML elements.
+It's also possible to have a generic custom component which accepts any React component, and HTML elements.
```

Comment:
nit: the term "HTML components" is probably fine just fine, but the React docs call these "built-in components" or "common components". See https://react.dev/reference/react-dom/components/common. In case you want to consider this:

```suggestion
It's also possible to have a generic custom component which accepts any React component, including [built-in components](https://react.dev/reference/react-dom/components/common).
```